build.gradle

```diff
@@ -505,10 +505,21 @@ def jmx = ["-Dcom.sun.management.jmxremote", "-Dcom.sun.management.jmxremote.por
 
 def logbackConfig = project.findProperty('logbackCfg') ?: "logback_prod.xml"
 
+def profilerTypeConfig = project.findProperty('profiler_type') ?: null
+
+def profilerOption = ""
+if( profilerTypeConfig.toString().equalsIgnoreCase('cpu')) {
+    profilerOption = jfr
+}
+else if (profilerTypeConfig.toString().equalsIgnoreCase('cpumem')){
```

Comment:
I'm unsure of how groovy handles null, but should the config default to an empty string instead of null?

Reply:
> > Will it work like that without any change to aws/build.gradle? How that parameter will be propagated from someone's machine to AWS?
> 
> I do not believe this will propagate to AWS - and will need to be added explicitly in other ways. If that is wanted then we should add a means explicitly.

yes it was not propagated in  AWS, But now its updated